### DISCUSSION TITLE
[Esercitazione 13 Dicembre 2023] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#321] [Esercizi - Viaggi/Piazza]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/ca666b69-2382-4fc7-9531-3c6df8ac008d)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/05135317-0b26-405b-87d1-5d0bdbe76dd5)

**Comment by Frisbon** [comment]
Analogo a @CiottoloMaggico 
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/f913a41c-3c06-4e82-8fce-f11b2adee37d)
